detect-nuget-inspector/detect-nuget-inspector/Inspection/Util/SolutionDirectoryPackagesPropertyLoader.cs

```diff
@@ -12,11 +12,17 @@ public class SolutionDirectoryPackagesPropertyLoader : PackageReferenceLoader
     {
 
         private String PropertyPath;
+        private HashSet<PackageId> RootCentrallyManagedPackages;
 
         public SolutionDirectoryPackagesPropertyLoader(String propertyPath)
         {
             PropertyPath = propertyPath;
         }
+        
+        public SolutionDirectoryPackagesPropertyLoader(String propertyPath, HashSet<PackageId> packageIds): this(propertyPath)
```

Comment:
Here and in other classes: it may be more useful to give arguments of constructors/methods more verbose names.
That way users of the class don't need to go into the actual class implementation to figure out what a given variable is for.